src/main/java/ch/wisv/payments/admin/DashboardController.java

```diff
@@ -19,12 +19,17 @@ public DashboardController(OrderService orderService) {
         this.orderService = orderService;
     }
 
-    @RequestMapping("/")
+    @RequestMapping("/admin")
```

Comment:
Why not just call it dashboard? I would also change the name of the page to dashboard.